tests/Dfe.PlanTech.Web.UnitTests/Controllers/QuestionsControllerTests.cs

```diff
@@ -26,8 +26,9 @@ public class QuestionsControllerTests
   private const string QUESTION_SLUG = "question-slug";
   private const string SECTION_SLUG = "section-slug";
   private const int ESTABLISHMENT_ID = 1;
-
-  private readonly Question _validQuestion = new Question()
+    private const string GET_NEXT_UNANSWERED_QUESTION_ACTION_NAME = "GetNextUnansweredQuestion";
```

Comment:
Can we lower constants please, This is old way of doing it.

Reply:
I'm old